src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/MvcViewFeaturesLoggerExtensions.cs

```diff
@@ -54,16 +56,21 @@ static MvcViewFeaturesLoggerExtensions()
                 1,
                 "Executing PartialViewResult, running view at path {Path}.");
 
-            _partialViewFound = LoggerMessage.Define<string>(
+            _partialViewFound = LoggerMessage.Define<string, double>(
                 LogLevel.Debug,
                 2,
-                "The partial view '{PartialViewName}' was found.");
+                "The partial view '{PartialViewName}' was found in {ElapsedMilliseconds}ms.");
 
             _partialViewNotFound = LoggerMessage.Define<string, IEnumerable<string>>(
                 LogLevel.Error,
                 3,
                 "The partial view '{PartialViewName}' was not found. Searched locations: {SearchedViewLocations}");
 
+            _partialViewResultExecuted = LoggerMessage.Define<string, double>(
+                LogLevel.Information,
+                4,
+                "Executed PartialViewResult - view at path {Path} in {ElapsedMilliseconds}ms.");
```

Comment:
Suggestion: `{Path}` -> `{FilePath}`

Reply:
I just realized that this has changed to view name instead. https://github.com/aspnet/Mvc/pull/7314#issuecomment-361668535
> I'd also expect the value /Views/Home/Index.cshtml to go with finding the view, and Index to go with the action results

Changing this to {ViewName} and the view finding to {FilePath}